istioctl upgrade feature docs (#5245)
istio/istio.io: content/en/docs/setup/upgrade/cni-helm-upgrade/index.md

---

> title: Upgrade using the CNI plug-in or Helm

Reviewer: ```suggestion
title: Upgrade using Helm
```

Reviewer: The CNI plug-in is an optional step in the upgrade, not an alternative to Helm

> description: Upgrade the Istio control plane and data plane independently using the CNI plug-in or Helm.

Reviewer: ```suggestion
description: Upgrade the Istio control plane and, optionally, the CNI plug-in using Helm.
```

Author: fixed

> description: Upgrade the Istio control plane and data plane independently using the CNI plug-in or Helm.

Reviewer: ```suggestion
description: Upgrade the Istio control plane using Helm.
```
